File: 01 Introdução/3224 Aaah/3224.c
# include <stdio.h>
# include <string.h>

int main(){
  char jon[1000];
  char medico[1000];
  scanf("%s", jon);
  scanf("%s", medico);
  if (strlen(medico) > strlen(jon)) printf("no\n");
  else printf("go\n");
}

// https://www.beecrowd.com.br/judge/pt/problems/view/3224
// Ranking 00115º, Time 0.000, Submiss Date 16/04/2023 12:07:24 (C99)

File: 01 Introdução/3224 Aaah/3224.java
import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    String jon = sc.nextLine();
    String medico = sc.nextLine();
    if (medico.length() > jon.length()) System.out.println("no");
    else System.out.println("go");
    sc.close();
  }
}


// https://www.beecrowd.com.br/judge/pt/problems/view/3224
// Ranking 00011º, Time 0.154, Submiss Date 16/04/2023 12:25:38 (Java 14)